src/core/search/search.cc

```diff
@@ -206,7 +218,7 @@ struct BasicSearch {
 
   IndexResult Search(const AstStarNode& node, string_view active_field) {
     DCHECK(active_field.empty());
-    return vector<DocId>{indices_->GetAllDocs()};  // TODO FIX;
+    return {&indices_->GetAllDocs()};
   }
```

Comment:
Most often KNN is used with the start node (all documents), so I want it to be a pure O(1) to fetch all documents. This is why I also added the `const DocVec*` variant